pandas/tests/indexes/timedeltas/test_tools.py

```diff
@@ -110,6 +110,16 @@ def test_to_timedelta_invalid(self):
             invalid_data, to_timedelta(invalid_data, errors="ignore")
         )
 
+        # GH#10720
```

Comment:
can you make a new test

Reply:
`timedeltas` passes normally. What logic should I test? (for now moved that test to other tests with `'coerce'`).